src/app/pages/overlays/complex-bottom-sheet/bottom-sheet/bottom-sheet.html

```diff
@@ -1,65 +1,60 @@
 <mat-nav-list>
     <div class="footer-list" data-cy="btm-sheet-sort">
-        <h3 class="mat-overline">Sort by</h3>
-        <div fxLayout="row wrap" fxLayoutAlign="center center">
-            <a [class.active]="activeSort == filterTypes.TIME" (click)="sortData(filterTypes?.TIME)">
-                <button mat-icon-button>
-                    <mat-icon mat-list-icon>access_time</mat-icon>
+        <div class="mat-h3">Sort By:</div>
```

Comment:
I think you got the labels backwards...Huayun wants us to _use_ the Overline label style in both React and Angular.

Reply:
Ah, okay! This is reverted now.